include/sp_vm_types.h

```diff
@@ -246,4 +249,26 @@ typedef struct sp_debug_symbol_s
 	sp_debug_symbol_raw_t *sym;	 /**< Pointer to original symbol */
 } sp_debug_symbol_t;
 
+/**
+ * @brief Context describing the VM state when the SPVM_DEBUGBREAK
+ * callback is called.
+ */
+typedef struct sp_debug_break_info_s
+{
+  uint16_t  version;   /**< Version of this struct */

```

Comment:
How is this version field supposed to be used?

Reply:
To know whether new fields are available. When adding more info to the struct, like maybe a pointer to the smx in memory to parse the debug symbols manually, allow extensions know which fields they can use.

Probably could get rid of this seperate version field and just increase the whole vm api version if it's changed.